Tests/SvgW3CTestRunner/RunTestsDialog.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
```

Comment:
I'm not going to review the GUI code, instead I just ran it and checked the outcome.
I think this is very helpful - both the separate runner (which also shows that some of the tests marked as "passing" are not really working), and the display of the difference in the original TestRunner.

Reply:
After testing with the #945 PR, I have decided to improve it.
- it is now modeless, and can be minimize.
- you can double-click or click a button to view the rendered images in the main TestRunner.
- you copy the selected file name or path to a clipboard.

Pending
- export the results so what we can compare the percentage, exception changes in each PR.

We will need to recreate the `passing-failing` lists after completing the PR merges and start work on the Issues.